WalletWasabi/Wallets/Wallet.cs

```diff
@@ -271,18 +271,15 @@ private async Task PerformFinalSynchronizationAsync(CancellationToken cancel)
 		{
 			using (await HandleFiltersLock.LockAsync(cancel).ConfigureAwait(false))
 			{
-				while (true)
-				{
-					cancel.ThrowIfCancellationRequested();
-					await PerformWalletSynchronizationAsync(SyncType.NonTurbo, cancel).ConfigureAwait(false);
-					if (LastProcessedFilter is { } lastProcessedFilter)
-					{
-						SetFinalBestHeight(new Height(lastProcessedFilter.Header.Height));
-					}
+				cancel.ThrowIfCancellationRequested();
```

Comment:
Is this line still needed?

If the cancellation token gets cancelled 
`await PerformWalletSynchronizationAsync(SyncType.NonTurbo, cancel).ConfigureAwait(false);` will throw OperationCancelledExc just the same, right?

Reply:
No it is not.